### DISCUSSION TITLE
Need help to report abuse

### DISCUSSION THREAD

**Discussion by Tstonehouse** [community/community#134103] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My ex is a software developer and is using developer tools to root and hack my accounts.    I need help to report and stop him.    Does anyone know how I can get to someone who can help me?

**Comment by Jesusavvillalba** [comment]
Change Passwords and Enable Two-Factor Authentication (2FA):

Immediately change the passwords on all your accounts to strong, unique passwords.
Enable two-factor authentication (2FA) on all accounts to add an extra layer of security.
Update and Secure Your Devices:

Ensure your devices are updated with the latest security patches.
Install and run reputable antivirus software to check for any malware or spyware.
Consider resetting your devices to factory settings if you suspect they have been compromised.
Contact the Service Providers:

Report the hacking incidents to the service providers of the compromised accounts (e.g., email, social media, banking).
Follow their procedures for securing your account and reporting abuse.
Gather Evidence:

Keep a record of any suspicious activity, such as unauthorized access or changes to your accounts.
Document any interactions or threats from your ex.
Report to Authorities:

Contact your local law enforcement agency to report the hacking and any other related threats or harassment.
Provide them with any evidence you've gathered.
Seek Legal Advice:

Consult with a lawyer who specializes in cyber law or domestic abuse. They can help you understand your rights and take legal action if necessary.
Support Resources:

Reach out to organizations that provide support for victims of cyber harassment and domestic abuse.
Consider talking to a counselor or support group to help you through this challenging time.

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @Tstonehouse. 

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
